### ISSUE TITLE
Surveys: Widget type picker in editor lacks aria-activedescendant

### ISSUE THREAD

**Issue by henrycatalinismith** [zetkin/app.zetkin.org#3642]
### The feature

When choosing a widget type for a question in the survey editor the lack of an `aria-activedescendant` attribute prevents assistive tech from informing disabled users which dropdown menu item is currently selected.

![Image](https://github.com/user-attachments/assets/d0417c2f-5f92-4f6e-8aec-d512702873cd)

### The bug

Screen readers are great at handling comboboxes. When focus is on an expanded combobox, you can press the up and down arrow keys to select the different options inside, and screen readers will tell you which option you've selected. We use `@mui` components in Zetkin which does a fantastic job of this.

The problem is we're rendering the wrong `@mui` component in the survey editor, and as a result the library is not managing the `aria-activedescendant` attribute right.

## Steps to reproduce

1. Create a new unpublished survey with a multiple choice question
2. Fire up VoiceOver
3. Focus the `Widget` input field on the multiple choice question

### Expected Behaviour

After opening the combox and using the up and down arrow keys to navigate around the options, the screen reader should announce the current selection thanks to `aria-activedescendant`.

https://github.com/user-attachments/assets/1638427b-dc0f-46ba-89b3-868954ab49b0

### Actual Behaviour

After opening the combox and using the up and down arrow keys to navigate around the options, the screen reader announces nothing.

https://github.com/user-attachments/assets/76bc4d3f-d289-49c4-ae3a-78da547b9213

## Diff

Here's how I produced the "Expected Outcome" screen recording. It's not a complete solution as it causes the visible label to disappear.

```diff
index c3b2ca9aa..284e14c27 100644
--- a/src/features/surveys/components/SurveyEditor/blocks/ChoiceQuestionBlock.tsx
+++ b/src/features/surveys/components/SurveyEditor/blocks/ChoiceQuestionBlock.tsx
@@ -13,6 +13,7 @@ import {
   IconButton,
   ListItemIcon,
   MenuItem,
+  Select,
   TextField,
   Typography,
 } from '@mui/material';
@@ -180,17 +181,13 @@ const ChoiceQuestionBlock: FC<ChoiceQuestionBlockProps> = ({
           variant="content"
         />
         {editable && (
-          <TextField
+          <Select
             fullWidth
             label={messages.blocks.choice.widget()}
-            margin="normal"
+            margin="none"
             onChange={(ev) => {
               setWidgetType(ev.target.value as WidgetTypeValue);
             }}
-            select
-            SelectProps={{
-              MenuProps: { disablePortal: true },
-            }}
```

We need to use `<Select>` here somehow.

**Comment by verpeccatum** [comment]
i will take a look at this

**Comment by twoheadGH** [comment]
Ich created PR https://github.com/zetkin/app.zetkin.org/pull/3725 to fix this issue.
